logging/monolog_email.rst

```diff
@@ -4,10 +4,9 @@
 How to Configure Monolog to Email Errors
 ========================================
 
-.. caution::
+.. versionadded:: 3.6
```

Comment:
Please add this line to `.doctor-rst.yaml` to the `whitelist`

Reply:
I did 👍

Reply:
thx @OskarStark :)